var/spack/repos/builtin/packages/py-torch-geometric/package.py

```diff
@@ -19,40 +19,54 @@ class PyTorchGeometric(PythonPackage):
     graphs as well as on 3D meshes or point clouds."""
 
     homepage = "https://github.com/rusty1s/pytorch_geometric"
```

Comment:
Not strictly a part of your changes, but based on the pypi page, the homepage should probably be https://github.com/pyg-team/pytorch_geometric